utilities/rpp-performancetests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -58,6 +58,17 @@ std::string get_interpolation_type(unsigned int val, RpptInterpolationType &inte
     }
 }
 
+std::string get_noise_type(unsigned int val)
```

Comment:
please use lut instead of conditional branch

Reply:
Will add this to the test suite PR